features/miam.feature

```diff
@@ -68,35 +68,35 @@ Feature: MIAM journey
 
 
   @happy_path
-  Scenario: Applicant did not attend a MIAM and has selected a misc valid reason
-    Then I should see "Have you attended a Mediation Information and Assessment Meeting (MIAM)?"
-    And I choose "No"
-    Then I should see "Do you have a valid reason for not attending a MIAM?"
-    And I choose "Yes"
-    Then I should see "Providing evidence of domestic abuse concerns"
-    And I check "None of these"
-    And I click the "Continue" button
-    Then I should see "Confirming child protection concerns"
-    And I check "None of these"
-    And I click the "Continue" button
-    Then I should see "Confirming why your application is urgent"
-    And I check "None of these"
-    And I click the "Continue" button
-    Then I should see "You’ve already been to a MIAM or are taking part in another form of non-court dispute resolution"
-    And I check "None of these"
-    And I click the "Continue" button
-    Then I should see "Confirming other valid reasons for not attending"
-    And I check "You can’t access a mediator"
-    And I check "There is no authorised family mediator with an office within 15 miles of your home"
-    And I click the "Continue" button
-
-    Then I should see "You don’t have to attend a MIAM"
-    Then I should see "Other exemptions"
-    Then I should see "There is no authorised family mediator with an office within 15 miles of your home"
-
-    And I click the "Continue" link
-    Then I should see "Safety concerns"
-    And the mediation changes end
+#  Scenario: Applicant did not attend a MIAM and has selected a misc valid reason
```

Comment:
I assume you comment this out because these test are not working and there is no way around it? Something Kalvin mentioned.